pages/[category]/index.tsx

```diff
@@ -1,56 +1,91 @@
 import { useRouter } from 'next/router'
-import Logo from 'components/logo/logo'
 import { sidebarData } from 'database/data'
 import { SubCategories } from 'types'
 import Link from 'next/link'
+import AddIcon from 'assets/icons/AddIcon'
+import ArrowIcon from 'assets/icons/ArrowIcon'
+import BreadCrumbs from 'components/Breadcrumps'
+import MessageIcon from 'assets/icons/MessageIcon'
 
 const CategoryPage = () => {
   const router = useRouter()
-  const { category } = router.query as { category: string}
+  const { category } = router.query as { category: string }
 
-  const subcategories: SubCategories[] = []
-  sidebarData.forEach((c) => {
-    if (c.category == category)
-      c.subcategory.forEach((a) => {
-        subcategories.push(a)
-      })
-  })
+  const subcategories: SubCategories[] = category
+    ? sidebarData.filter((c) => c.category == category)[0]['subcategory']
+    : []
 
   return (
     <section className="flex min-h-[calc(100vh-165px)] flex-col">
       <div className=" m-auto items-start gap-2">
         {subcategories?.length ? (
           <>
-            <div className="m-auto gap-2 flex flex-col items-center justify-center ">
-              <p className="md:text-4xl text-xl uppercase font-bold text-theme-secondary dark:text-gray-text w-fit mx-auto text-center">
-                <span className="text-theme-primary">-/</span>
-                {category.split('-').join(' ')}
-              </p>
-              <p className="text-xl hidden md:block">
-                Get access to all exclusive{' '}
-                <span className="capitalize text-theme-primary">
-                  {category.split('-').join(' ')}
-                </span>{' '}
-                resources!
-              </p>
+            <div className="flex mt-8 justify-between	">
```

Comment:
Please, avoid unnecessary spaces.

```suggestion
            <div className="flex mt-8 justify-between">
```